config/groups.yaml

```
@@ -578,3 +578,9 @@
     - "2614670"
     - "2603372"
   id: 190031
+  - name: "SEPP Assignment Group"
+  members:
+    - "2629286"
```

This isn't valid YAML, it would need to be:
```suggestion
- name: "SEPP Assignment Group"
  members:
    - "2629286"
```